server.js

```diff
@@ -1,29 +1,91 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import mongoose from 'mongoose';
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+// Model for each message object
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlenght: 140,
+  },
+  hearts: {
+    type: Number,
+    default: () => 0,
+  },
+  createdAt: {
+    type: Date,
+    default: () => Date.now(),
+  },
+});
 
 // Defines the port the app will run on. Defaults to 8080, but can be 
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
+
+const documentation = {
+  "Welcome": "Welcome to Claire's Happy Thoughts API 🌼",
+  "GET endpoint": {
+    "https://claireshappythoughts.herokuapp.com/thoughts": "Use this endpoint in your application to show up to the last 20 thoughts posted",    
+  },  
+  "POST endpoint 1": {
+    "https://claireshappythoughts.herokuapp.com/thoughts": "Use this endpoint in your application to allow for the user to post a thought to the database. Only allows for the message to be sent in the body of the post request.",
+  },
+  "POST endpoint 2": {
+    "https://claireshappythoughts.herokuapp.com/thoughts/:thoughtId/like": " Use this endpoint to allow for users to like thoughts by including the thought id in the url which targets the thought with that specific id and then updates the heart count of that thought in the database.",
+  },
+};
```

Comment:
I really like this part of your code, so clean and understandable! Maybe you can separate to different lines to be more readable, but that is just a tip!(: